File: CON BDD/EUREKABANK_SOAP_DOTNET_GR06/08. SERVIDOR/Modelo/ClienteResumen.cs
using System.Runtime.Serialization;

namespace SERVIDOR.Modelo
{
    [DataContract]
    public class ClienteResumen
    {
        [DataMember]
        public string Codigo { get; set; }

        [DataMember]
        public string Dni { get; set; }

        [DataMember]
        public string Nombre { get; set; }

        public ClienteResumen()
        {
            Codigo = string.Empty;
            Dni = string.Empty;
            Nombre = string.Empty;
        }
    }
}

File: CON BDD/EUREKABANK_SOAP_JAVA_GR06/07. CLIMOVIL/eurekabank_soap_java_climov_gr06/app/src/main/java/com/example/eurekabank_soap_java/modelo/ClienteResumen.java
package com.example.eurekabank_soap_java.modelo;

/** Resumen de cliente (para el admin). */
public class ClienteResumen {
    private String codigo;
    private String dni;
    private String nombre;

    public String getCodigo() { return codigo; }
    public void setCodigo(String v) { this.codigo = v; }
    public String getDni() { return dni; }
    public void setDni(String v) { this.dni = v; }
    public String getNombre() { return nombre; }
    public void setNombre(String v) { this.nombre = v; }

    @Override public String toString() {
        return codigo + " · DNI " + dni + " · " + nombre;
    }
}
